python/paddle/fluid/tests/unittests/mkldnn/test_pool2d_bf16_mkldnn_op.py

```diff
@@ -15,16 +15,57 @@
 from __future__ import print_function
 
 import unittest
-import os
 import numpy as np
 import paddle.fluid.core as core
-from paddle.fluid.tests.unittests.op_test import OpTest, skip_check_grad_ci, convert_float_to_uint16
-from paddle.fluid.tests.unittests.test_pool2d_op import TestPool2D_Op, avg_pool2D_forward_naive, max_pool2D_forward_naive
+from paddle.fluid.tests.unittests.op_test import OpTest, OpTestTool, skip_check_grad_ci, convert_float_to_uint16
```

Comment:
I don't see "skip_check_grad_ci" used in this file, I think that we can delete it